solutions/swb-reference/integration-tests/tests/multiStep/dataset.test.ts

```diff
@@ -45,7 +46,7 @@ describe('multiStep dataset integration test', () => {
       envTypeConfigId: settings.get('envTypeConfigId'),
       envType: settings.get('envType'),
       datasetIds: [dataSet.id],
-      name: uuidv4(),
+      name: `${resourceTypeToKey.dataset.toLowerCase()}${uuidv4()}`,
```

Comment:
do we need the `-` here that is present in other places?

Reply:
Oops, this is just the name of the DS. It can be any value. I'm going to use `randomTextGenerator.getFakeText('dataset-name')` so it's clear that this field doesn't have to follow the id format.